config.yaml

```diff
@@ -22,14 +22,19 @@ logging:
 
 # Statsd metrics reporter
 metrics:
+  enable_prometheus: true
+  prometheus:
+    port: 9000
```

Comment:
Hm.. this doesn't feel great either, sorry. How about that:

metrics could be an array or an object(b-c reasons). If it's an object - treat it as a array with 1 element.

Each element describes a metrics type. So if we have both statsd and prometheus, we can have smith like 

```
metrics:
 - type: statsd
    host: localhost
    port: 8125
 - type: prometheus
    port: 9000
```

Also, we can log a deprecation warning if the config is an object.

Reply:
Took a shot at this.  Let me know what you think.